addons/point_of_sale/static/src/app/customer_display/customer_display_template.xml

```diff
@@ -160,7 +160,7 @@
                                     </t>
                                     <t t-if="orderline.get_customer_note()">
                                         <li class="product_details">
-                                            <i class="fa fa-sticky-note" role="img" aria-label="Customer Note" title="Customer Note" />
+                                            <i class="fa fa-sticky-note" role="img" aria-label="Item Note" title="Item Note" />
```

Comment:
Use 'fa-sticky-note-o' as specified in the spec

Reply:
@mobt-odoo In the spec it is only mentioned for Footer Note. Do we need to do it for Item Note too.
If yes then could you please update the spec accordingly.

Thanks!.